balena/settings.py

```diff
@@ -31,14 +31,19 @@ class Settings(object):
     CONFIG_FILENAME = 'balena.cfg'
     DEFAULT_SETTING_KEYS = set(['pine_endpoint', 'api_endpoint', 'api_version',
                                 'data_directory', 'image_cache_time',
-                                'token_refresh_interval', 'cache_directory', 'timeout'])
+                                'token_refresh_interval', 'cache_directory', 'timeout',
+                                'image_maker_endpoint', 'image_maker_endpoint_version',
+                                'device_actions_endpoint_version'])
 
     _setting = {
         # These are default config values to write default config file.
         # All values here must be in string format otherwise there will be error when write config file.
         'pine_endpoint': 'https://api.balena-cloud.com/v5/',
         'api_endpoint': 'https://api.balena-cloud.com/',
         'api_version': 'v5',
+        'image_maker_endpoint': 'https://img.balena-cloud.com/api/v1/',
+        'image_maker_endpoint_version': 'v1',
```

Comment:
This seems not being used.
Should we remove the `v1` from the `image_maker_endpoint` and change the implementation or just drop this option?

Reply:
Dropped this option.